pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -13,6 +13,7 @@
 , pkgsBuildTarget
 , pkgsHostTarget
 , targetPackages
+, fetchpatch
```

Comment:
`error: duplicate formal function argument 'fetchpatch'`

We already have it below.